python_modules/libraries/dagster-looker/dagster_looker/lkml/asset_utils.py

```diff
@@ -1,11 +1,104 @@
 from pathlib import Path
-from typing import List, Sequence
+from typing import AbstractSet, Any, List, Mapping, Sequence, Tuple
 
 import lkml
 import yaml
 from dagster import AssetSpec
+from dagster._core.utils import toposort_flatten
+
+from dagster_looker.lkml.dagster_looker_lkml_translator import (
+    DagsterLookerLkmlTranslator,
+    LookMLStructureType,
+)
+
+
+def deep_merge_objs(onto_obj: Any, from_obj: Any) -> Any:
```

Comment:
Ellipsis usage in this function is... unusual (I like it!). 
Could you write a comment explaining this choice?

Reply:
I am using it as a sentinel value for "no value", specifically to handle the case of explicitly overriding with `None` vs not specifying an override

For example, merging the two dicts:
`{"a": 1, "b": 2}` and `{"a": None}`, we'd like the result to be `{"a": None, "b": 2}`. In this case, the recursive `deep_merge_objs` will be called with `deep_merge_objs(1, None) -> None` and `deep_merge_objs(2, ...) -> 2`.

Reply:
Oh, understood! I thought of something like this but couldn't figure out the exact use case. 
Would you mind leaving a comment about this in code? For future readers.